charts/rancher-gatekeeper/100.1.0+up3.7.1/Chart.yaml

```diff
@@ -2,11 +2,11 @@ annotations:
   catalog.cattle.io/auto-install: rancher-gatekeeper-crd=match
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: OPA Gatekeeper
-  catalog.cattle.io/kube-version: '>= 1.16.0-0'
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.25.0-0'
```

Comment:
This was released in Rancher 2.6, which supports up to 1.23

https://github.com/rancher/rancher/issues/37376

https://www.suse.com/assets/EN-2.6.5SupportMatrix-300422-0116-26.pdf

Are we sure this works on 1.24?